pkg/controller/jobs/job/job_controller.go

```diff
@@ -275,6 +275,15 @@ func (j *Job) Finished() (metav1.Condition, bool) {
 	return condition, finished
 }
 
+// PriorityClass calculates the priorityClass name needed for workload according to the following priorities:
+//  1. .Spec.Template.Spec.PriorityClassName
+func (j *Job) PriorityClass() string {
+	if len(j.Spec.Template.Spec.PriorityClassName) != 0 {
+		return j.Spec.Template.Spec.PriorityClassName
+	}
+	return ""
+}
```

Comment:
Why add this? It works without it.

Reply:
@Gekko0114 The `JobWithPriorityClass` interface is optional. So, all jobs don't need to implement the interface.

Reply:
I misunderstood. I've removed this function. Thank you.